### PR TITLE
add Romanian language to the app

### PR DESCRIPTION
**Pull request by augustin-colesnic**
## Romanian Language
Adds Romanian language to the app

### CODE REVIEW

**Comment by tobias-klein** [comment]
Excellent, thanks!

Feel free to add another change before I merge - adding your name to the list of contributors here - this will show up in the info dialog:
https://github.com/ezra-bible-app/ezra-bible-app/blob/master/app/frontend/components/info_popup.js#L106

Just push another commit ... that should work and then show up here.

**Comment by tobias-klein** [comment]
And another question: Is it OK for you to contribute updates to this translation roughly every two months? That's the release cycle of Ezra Bible App. Whenever a new release is being prepared I open an issue tagging all the translators (roughly two weeks before release). There are usually a few new/updated strings every release. I usually prepare a list of changes, so the update should be fairly easy (see #178 as example).

**Comment by baakaweb** [comment]
> Excellent, thanks!
> 
> Feel free to add another change before I merge - adding your name to the list of contributors here - this will show up in the info dialog:
> https://github.com/ezra-bible-app/ezra-bible-app/blob/master/app/frontend/components/info_popup.js#L106
> 
> Just push another commit ... that should work and then show up here.

Sure, thing! Welcome. I will!

**Comment by baakaweb** [comment]
> And another question: Is it OK for you to contribute updates to this translation roughly every two months? That's the release cycle of Ezra Bible App. Whenever a new release is being prepared I open an issue tagging all the translators (roughly two weeks before release). There are usually a few new/updated strings every release. I usually prepare a list of changes, so the update should be fairly easy (see #178 as example).

Yes, and not only translations. I like this project and I also want to contribute on other issues as well.  :)

**Comment by baakaweb** [comment]
Screenshot 2021-05-22 at 22 46 35
Screenshot 2021-05-22 at 22 44 55
Screenshot 2021-05-22 at 22 44 34

Hi @tobias-klein Could you look into why some toolbar buttons are not translated, as Search button and New Tag

**Comment by baakaweb** [comment]
Screenshot 2021-05-22 at 22 58 41
@tobias-klein  after I closed the dev app and run again npm  start all was fine with the buttons as can be seen in the screenshot. Peace out! :)

File: locales/ro/translation.json
Comment (tobias-klein) [inline locales/ro/translation.json:51]: I would suggest to keep the product name "Ezra Bible App" unmodified wherever it is used.

File: locales/ro/translation.json
Comment (tobias-klein) [inline locales/ro/translation.json:69]: Same comment as above

File: locales/ro/translation.json
Comment (tobias-klein) [inline locales/ro/translation.json:102]: The term Strong's is a unique concept, I suggest to keep that

File: locales/ro/translation.json
Comment (tobias-klein) [inline locales/ro/translation.json:103]: See above

File: locales/ro/translation.json
Comment (tobias-klein) [inline locales/ro/translation.json:130]: See above

File: locales/ro/translation.json
Comment (tobias-klein) [inline locales/ro/translation.json:251]: See above

File: locales/ro/translation.json
Comment (tobias-klein) [inline locales/ro/translation.json:252]: See above

File: locales/ro/translation.json
Comment (tobias-klein) [inline locales/ro/translation.json:256]: See above

**Review by tobias-klein — CHANGES_REQUESTED** [review]
I have a couple more content-related review comments! It is mostly about keeping the product name _Ezra Bible App_ intact.

File: locales/ro/translation.json
Comment (augustin-colesnic) [reply locales/ro/translation.json:51]: ok, i will do the change

File: locales/ro/translation.json
Comment (augustin-colesnic) [reply locales/ro/translation.json:102]: Isn't  '_Strong's_' concept just an index of words to be explained? If you would like to keep the author name in this (James Strong) then what about '**Concordanța lui Strong**' = '**Strong's Concordance**' otherwise i will leave this untranslated as it has no understanding in Romanian.

File: locales/ro/translation.json
Comment (augustin-colesnic) [reply locales/ro/translation.json]: done

File: locales/ro/translation.json
Comment (augustin-colesnic) [reply locales/ro/translation.json]: done

File: locales/ro/translation.json
Comment (augustin-colesnic) [reply locales/ro/translation.json]: done

File: locales/ro/translation.json
Comment (augustin-colesnic) [reply locales/ro/translation.json:103]: In Romanian si a lot better if I would translate something as:  e.g-  **Strong's Hebrew Concordance**. So I am waiting for a decision from you.

File: locales/ro/translation.json
Comment (tobias-klein) [reply locales/ro/translation.json:102]: Yes to what you suggested. The specific numbering system is from this guy named Strong and it is unique, not just a generic concept.

File: locales/ro/translation.json
Comment (tobias-klein) [reply locales/ro/translation.json:103]: Maybe you could put Strong's in parentheses behind your translation?

Comment (augustin-colesnic) [reply locales/ro/translation.json:103]: Yes, for now that will do :) 
Let's see in the future other suggestions from community on any language.

File: locales/ro/translation.json
Comment (augustin-colesnic) [reply locales/ro/translation.json]: done

File: locales/ro/translation.json
Comment (augustin-colesnic) [reply locales/ro/translation.json]: pushed changes

**Review by tobias-klein — APPROVED** [review]

**Comment by tobias-klein** [comment]
Thank you once more for all your work on this @augustin-colesnic! Going to merge now 🙂

**Pull request merged**
